### ISSUE TITLE
Cannot install from release tar ball due to autogen

### ISSUE THREAD

**Issue by morxa** [0k/shyaml#37]
Installing from the release tarball fails because `autogen.sh` needs the git repository for version information.

To reproduce:

1. Download the release tar ball
1. Run `./autogen.sh` in the extracted tar ball. This will fail with the following error:

        autogen.sh: error: Didn't find a git repository (or no tags found).  ``./autogen.sh`` uses git to create changelog and version information.

It would be nice if installing would work without cloning the complete repository.

I'm currently considering to package shyaml as a Fedora package. However, I cannot clone a repository while building a package, I need to be able to use the tarball. So while this might be a small issue for a normal user, it's a blocker for the Fedora package.

**Comment by vaab** [comment]
Yes, the default packages provided by github are not compatible with ``autogen.sh``. I had the intention to invest time in providing real non-git source packages (and others). Would this help ?

**Comment by morxa** [comment]
Yes, that would work!

**Comment by vaab** [comment]
Ok, I have a somewhat tight schedule, I am not sure of how many time I have in front of a computer in the next days. I'd like to automate the github release process for all my projects in the same move, to ensure also a continuous and coherent flow of release.

**Comment by morxa** [comment]
It's been a while. What's the current status?

**Comment by vaab** [comment]
Sorry, it won't be implemented for now, still under heavy load. But it is related it #27. I still want to do that.
